snippets/resize_header_on_scroll/src/custom_css.css

```diff
@@ -0,0 +1,26 @@
+/* I only use this on bigger screens to save real estate on smaller ones. I made this for the Bedford Theme. Other themes sometime use different ID and class names. */
+
+@media only screen and (min-width: 641px){
+#header {
+	position: fixed; /*this keeps your header visible even on scroll.*/
+	transition: all 0.5s;
+}
+#logoImage img, #headerNav nav a {
+  transition: all 0.5s;
+}
+.banner-thumbnail-wrapper {
+	margin-top: 83px; /*this keeps the banner from being hidden behind the fixed header, should be equal to the height of the fixed header.*/
+}
+header.smaller {
+	height: 65px; /*this is the height you want your header to be after scrolling.
```

Comment:
Hey @fairmorningblue -- there is a small bug here, as the comment needs to be terminated. Note how the following two lines are "commented out".

To fix, just append `*/` to this line, commit, and re-push. Then I can merge this in!

So, line becomes:

```
height: 65px; /*this is the height you want your header to be after scrolling.*/
```